routing-daemon/lib/openshift/routing/controllers/asynchronous.rb

```diff
@@ -342,13 +342,13 @@ def create_monitor monitor_name, path, up_code, type, interval, timeout
       monitors.push monitor_name
     end
 
-    def delete_monitor monitor_name, pool_name=nil
+    def delete_monitor monitor_name, pool_name=nil, type
```

Comment:
Ruby won't accept this because you cannot have a non-optional parameter after an optional parameter.  We'll need to drop `=nil` from `pool_name` to make it non-optional.